src/amber/cli/commands/deploy.cr

```diff
@@ -35,7 +35,7 @@ module Amber::CLI
         (spinner = Spin.new(0.1, Spinner::Charset[:arrow2].map { |c| c.colorize(:light_green).to_s })).start
         shard = YAML.parse(File.read("./shard.yml"))
         @project_name = shard["name"].to_s
-        @server_name = "#{project_name}-#{args.server_suffix}".gsub(/[^\w\d\-]|_/, "")
+        @server_name = "#{project_name}-#{args.server_suffix}".gsub(/[^\w\d\-]/, "")
```

Comment:
It seems like this regex should be: `/[^\w\d|_-]/`? 

Actually, this is strange because `"#{project_name}-#{args.server_suffix}"` has a `-` in it, which the gsub is just going to delete.

Reply:
I also think it should replace the findings with `-` to make it readable.